querybook/webapp/components/AIAssistant/AICommandInput.tsx

```diff
@@ -139,6 +196,38 @@ export const AICommandInput: React.FC<AICommandInputProps> = forwardRef(
             </div>
         );
 
+        const loadTables = useCallback(
+            (
+                keywords: string,
+                callback: (
+                    options: Array<{ id: string; display: string }>
+                ) => void
+            ) => {
+                if (!keywords) {
+                    return;
+                }
+
+                SearchTableResource.searchConcise({
+                    metastore_id: metastoreId,
+                    keywords,
+                }).then(({ data }) => {
+                    const filteredTableNames = data.results.filter(
+                        (result) =>
+                            !mentionedTables.includes(
+                                `${result.schema}.${result.name}`
+                            )
+                    );
+                    const tableNameOptions = filteredTableNames.map(
+                        ({ id, schema, name }) => ({
```

Comment:
id is not used here?

Reply:
i originally thought it would be used in line 222, but it would have been `[id]: ...`

Reply:
was meant to use it, but found not necessary, forgot to remove it.